package.json

```
@@ -41,10 +41,11 @@
     "dev:invoke": "(cd packages/airnode-node && yarn run dev:invoke)",
     "dev:list": "(cd packages/airnode-operation && yarn run dev:list)",
     "dev:stop": "(cd packages/airnode-operation && yarn run dev:stop)",
-    "docker": "yarn docker:artifacts && yarn docker:deployer && yarn docker:node",
-    "docker:artifacts": "docker rmi --force api3/airnode-artifacts && docker build --tag api3/airnode-artifacts:latest --file docker/Dockerfile .",
-    "docker:deployer": "docker build --tag api3/airnode-deployer:latest --file packages/airnode-deployer/docker/Dockerfile .",
-    "docker:node": "docker build --tag api3/airnode-client:latest --file packages/airnode-node/docker/Dockerfile .",
+    "docker": "yarn docker:build:artifacts && yarn docker:build:deployer && yarn docker:build:node && yarn docker:build:admin",
+    "docker:build:admin": "docker build --tag api3/airnode-admin:latest --file packages/airnode-admin/docker/Dockerfile .",
+    "docker:build:artifacts": "docker rmi --force api3/airnode-artifacts && docker build --tag api3/airnode-artifacts:latest --file docker/Dockerfile .",
```

This one isn't building 😉 
```suggestion
    "docker:rm:artifacts": "docker rmi --force api3/airnode-artifacts && docker build --tag api3/airnode-artifacts:latest --file docker/Dockerfile .",
```

---

Nevermind, I just saw the second part 🤦🏻‍♂️

---

It is.

I mean it's also removing the old container, but it also does the fresh build